### PR TITLE
Reworded first paragraph for clarity.

### PR DESCRIPTION
**Pull request by huntley**
@inoxx03 @tradej Please review. Thanks!

### CODE REVIEW

**Comment by inoxx03** [comment]
test

**Comment by inoxx03** [comment]
add to whitelist

File: docs/topics/proc_configuring-your-vertx-application-to-use-agroal.adoc [hunk 1]
```diff
@@ -2,8 +2,7 @@
 = Configuring your {VertX} application to use Agroal
 
 // hard-coded version, beacuse it does not change
-Starting with {VertX} release `3.5.1.redhat-003`, Agroal is the default JDBC connection pool, replacing C3P0 used as the default option in previous releases.
-Due to differences in property names between C3P0 and Agroal, upgrading to a newer release of {VertX} might break the JDBC connection pool configuration of your existing {VertX} applications.
+Starting with {VertX} release `3.5.1.redhat-003`, Agroal replaced C3PO as the default JDBC connection pool. C3PO and Agroal use different property names and upgrading to a newer release of {VertX} might break the JDBC connection pool configuration of your {VertX} applications.
```
Comment (Ladicek) [inline docs/topics/proc_configuring-your-vertx-application-to-use-agroal.adoc]: Multiple sentences on a single line ;-)

**Comment by tradej** [comment]
add to whitelist

**Pull request merged**